### PR TITLE
fixes for requestShippingRates populates wrong store and website IDs …

### PR DESCRIPTION
**Pull request by sheetal2jcommerce**
### Description (*)
Fixes applied when creating a new order in the admin panel, clicking "Get shipping methods and rates" shipping methods returned for the chosen store.If the instance configured for multipile websites,all attributes which require website specific data like currency, this fixes will work.

### Fixed Issues (if relevant)

1. magento/magento2#15497: requestShippingRates populates wrong store and website IDs when retrieving shipping rates and methods in admin panel

### Manual testing scenarios (*)

1. Create two or more websites within the Magento install, and create each with their own store and store views.
2. Turn on some shipping methods for one store, and other shipping methods for the other store.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=18560) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @sheetal2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Quote/Model/Quote/Address.php [hunk 1]
```diff
@@ -999,17 +999,24 @@ public function requestShippingRates(\Magento\Quote\Model\Quote\Item\AbstractIte
 
         $request->setFreeMethodWeight($item ? 0 : $this->getFreeMethodWeight());
 
+        
+         // =======================================================
```
Comment (sivaschenko) [inline app/code/Magento/Quote/Model/Quote/Address.php:1003]: I don't think this prominent code fragment separation is necessary. Please remove extra lines and unnecessary comments before and after the changed code fragment to be consistent with the rest of the codebase.

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Only a small cosmetic change required

**Comment by sivaschenko** [comment]
@sheetal2jcommerce thank you for the contribution! Please resolve the code review comment so that we can process this pull request further!

File: app/code/Magento/Quote/Model/Quote/Address.php (hunk 1 above)
Comment (sheetal2jcommerce) [reply app/code/Magento/Quote/Model/Quote/Address.php:1003]: Hi Sivaschenko,
  Updating code as per ur suggession.

Thanks
Sheetal

Comment (sivaschenko) [reply app/code/Magento/Quote/Model/Quote/Address.php:1003]: Hi @sheetal2jcommerce , sorry, I don't see the update. No new commits are added to the branch...

**Comment by sivaschenko** [comment]
@sheetal2jcommerce I see that you created a new pull request for the requested changes. You can actually push changes to the branch without the need to create new pull requests, new commits will be included within pull request automatically.

However, since you already created a new pull request, please close this one as a duplicate.

**Comment by sivaschenko** [comment]
New pull request: https://github.com/magento/magento2/pull/19126

**Comment by sivaschenko** [comment]
Closing as the work on these changes continue in https://github.com/magento/magento2/pull/19126

**Pull request closed without merging**